### ISSUE TITLE
Orthogonal initialization gives inaccurate result when running on GPU 

### ISSUE THREAD

**Issue by dargilboa** [keras-team/keras#11120]
When generating an orthogonal matrix using 'tf.keras.initializers.Orthogonal' and running on a GPU, if the matrix is of size larger than nxn for n of about 1000 the resulting matrix is not orthogonal. The deviations are several orders of magnitude larger than expected from finite precision errors. 
The issue doesn't occur for smaller n (say of size 100) or when running on a CPU. 
The issue occurred when using both a Telsa V100 and a Tesla K80, with Keras 2.2.2 and Tensorflow 1.10.1.

In the example below, the final result should be an identity matrix to within numerical precision error, but has deviations of order 1e-2 in the individual entries. 

`import tensorflow as tf`
`tf.enable_eager_execution()`
`tf.set_random_seed(0)`
`init = tf.keras.initializers.Orthogonal(1, seed=0)`
`a = init([1000,1000])`
`tf.matmul(a,tf.transpose(a))`

**Comment by tazr** [comment]
This issue seems to be gone in tf 1.13.1 / keras 2.2.4-tf.
